src/polar/PolarGrid.tsx

```diff
@@ -14,7 +14,7 @@ interface PolarGridProps {
   polarAngles?: number[];
   polarRadius?: number[];
   gridType?: 'polygon' | 'circle';
-  radialLines: boolean;
+  radialLines?: boolean;
```

Comment:
this is defaulted so this change is fine, tests need this to be optional as it isn't specified in some tests